packages/gatsby/src/utils/websocket-manager.ts

```diff
@@ -190,7 +191,9 @@ export class WebsocketManager {
         let pageData = this.pageResults.get(path)
         if (!pageData) {
           try {
-            pageData = await getPageData(path)
+            pageData = process.env.GATSBY_EXPERIMENTAL_GET_PAGE_DATA
```

Comment:
Do you think this should temporary flag that we will completely replace with whatever method of feature flagging query on demand will do? Or would we want to keep this in addition to query on demand check?

Reply:
Yeah, I think using the same flag as queries on demand makes perfect sense. I can change it to whatever env variable you choose for QoD.